Create CCC-TAC-Minutes-2023-05-04.md (#168)
confidential-computing/governance: TAC/Meetings/2023/2023-05-04/CCC-TAC-Minutes-2023-05-04.md

* DT posted the link to the [Growth Plan document](https://github.com/confidential-computing/governance/blob/main/growth-plans.md)
* Eric Voit (EV) posted the link to the [CNCF definitions](https://www.cncf.io/projects/)
* EV commented that the stages text was initially copied from the CNCF definitions
* Mona Vij (MV) asked anyone using Gramine to notify the project and to be added to the Github page
* Discussion ensued on community participation and commit quality, more details need to be added to the graduation definition
* DT added in chat - The graduation stage requirements draft text currently says: "Have a defined governing body of at least 5 or more members (owners and core maintainers, or similar technical role), of which no more than 1/3 is affiliated with the same employer."
* DT brought up that according to the [Gramine CONTRIBUTING document](https://github.com/gramineproject/gramine/blob/master/CONTRIBUTING.rst#management-team-maintainers), Gramine does not meet the requirements for Graduation.
* DM discussed adding the recommendation for the project maintainers to take the LF training on DEI (link above)

> * Michal Kowalczyk asked how does a company being funded by another company be documented? for example is the Invisible Things company different from Intel?

Reviewer: ```suggestion
* Michal Kowalczyk asked how is a company being funded by another company treated? for example is the Invisible Things company different from Intel?
```
grammar

**TAC Goal Topic: Education**

* Cfir Cohen discussed the Outbound Education topic
  * At RSA conference, the speaker asked how many of the 12 people had heard of Confidential Computing, 2 people raised their hands, demonstrating the gap in Outbound Education
  * Review of White papers, blog posts
  * CCS talks planned
  * Other ideas: Mike Bursell expressed interest and volunteered to help however possible
  * Meet customers where they are - go to industry conferences that are not only security focused, discussion ensued
  * Sync with the similar efforts in the Outreach committee
  * DT asked about the "As measured by" bullets in the TAC Goal Topic slide - "We promoted" isn't "measured" per se, are there good ways to actually measure the promotion?
  * CC talked about Use cases - tie to technology offerings - also overlaps with Outreach goals
  * MV asked if we have engaged regulatory bodies, DM discussed where this is covered with interfacing with NIST and the work in the GRC Sig

> **Update: Attestation SIG - Shanwei Cen, Greg Kostal, Keith Moyer - deferred to next meeting

Reviewer 2: formatting: there's probably a missing `**` on this line

**Meeting Minute Approval**

Proposed: That the minutes of the April 20, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.

* Quorum was reached and the minutes were approved by the voting members of the TAC. 

**Tech Talk: VirTEE**

* Tyler Fanelli (TF) and Larry Dewey (LD) - [VirTEE project website](https://virtee.io/)
* Discussed libraries and tools - confidential workloads and containers
* Goal of the project - Make it easier to develop confidential virtualization, lower barriers to entry, not tied to a specific hardware architecture

> * DT asked if the [KBS protocol spec](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md), have they considered submitting to the IETF RATS Working Group for review? LD agreed that would be a good discussion

Reviewer: ```suggestion
* DT asked about the [KBS protocol spec](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md), have they considered submitting to the IETF RATS Working Group for review? LD agreed that would be a good discussion
```

* MB asked about attestation 
* Thomas Fossati asked about defining an API for interacting with verifiers, perhaps a RESTful API, TF agreed to interact with VirTEE
* LD added that they are working toward sandbox stage as a new project for the CCC

> * DT asked about SEV and SMP - supported and promoting

Reviewer: ```suggestion
* DT asked about SEV vs SEV-SNP - only the latter appears in a CCC whitepaper
```
